recipes/tadrep/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: tadrep
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/oschwengers/tadrep/archive/v{{ version }}.tar.gz
+  sha256: '5dcba6aa04e4e87d985779dd6c7968a20550ebb1304defc59f66bba698239079'
+
+build:
+  noarch: python
```

Comment:
please use noarch: generic

It does contain non python dependencies

Reply:
Are runtime deps also taken into account? I thought this only relates to build deps? TaDReP itself is pure Python, therefore I thought `noarch: python` would be the right tag.